portfolio/src/main/java/com/google/sps/servlets/PostCommentServlet.java

```diff
@@ -39,12 +39,13 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment-input");
     if (!text.isBlank()) { 
+      text = text.trim();
```

Comment:
```suggestion
    String text = request.getParameter("comment-input").trim();
    if (!text.isBlank()) { 
```
Maybe? If we don't want to post empty comments